scripts/create_build_web.py

```diff
@@ -126,5 +144,7 @@ def run(version):
     version = get_version()
     parser = argparse.ArgumentParser()
     parser.add_argument("--version", help="new version", default=version)
+    parser.add_argument("--browser", help="browser name", default=sys.argv[1])
```

Comment:
add choices (chrome or firefox) and check validity

Reply:
fixed